nixos/doc/manual/installation/installing-usb.xml

```diff
@@ -23,7 +23,7 @@ $ diskutil list
 [..]
 $ diskutil unmountDisk diskN
 Unmount of all volumes on diskN was successful
-$ sudo dd bs=1m if=nix.iso of=/dev/rdiskN
+$ sudo dd bs=1000000 if=nix.iso of=/dev/rdiskN
```

Comment:
might be better to just suggest `cp`: `sudo cp nix.iso /dev/rdiskN`.

Reply:
oh cool! Didn't know. Does it work for darwin as well?

Reply:
not 100% sure, @LnL7 ?

Reply:
You can't *control* cp's buffer size, which might be relevant to performance here. `dd`, when used with a byte-size is more universal. Though here the instructions are **darwin-specific**. Do we know whether all expected darwin `dd` fail with `1m`?

Reply:
darwin dd supports 1m but some users have coreutils dd installed through Nixpkgs. I don't think there's really a way to support both without just avoiding the "M" part altogether.

Reply:
If someone can verify the cp command, that would be fine with me.

Reply:
Using cp seems to work but fails at the end, this can be avoided with `-X`  but that's a bsd/apple specific flag so it has the same problem as `dd bs=1m` vs `dd bs=1M`.

```
cp: could not copy extended attributes to /dev/rdisk3: Operation not permitted
```